pages/index.js

```diff
@@ -46,9 +47,13 @@ const Index = () => (
       <div style={{ height: '150px' }} />
       <RyrSectionConnector fill="#5AAD54" bg="#DDFDDF" />
     </section>
+    <section style={{ backgroundColor: '#DDFDDF' }} />
+
     <section style={{ backgroundColor: '#DDFDDF' }}>
-      <div style={{ height: '150px' }} />
+      <RyrInstructions />
+      <div style={{ height: '20px' }} />
```

Comment:
As there is content in this section, you do not need to setup the height here.

Reply:
Fixed this section